code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -338,3 +347,21 @@
 	// Add verb for re-opening the interview panel, fixing chat and re-init the verbs for the stat panel
 	add_verb(src, /mob/dead/new_player/proc/open_interview)
 	add_verb(client, /client/verb/fix_tgui_panel)
+
+///Resets the Lobby Menu HUD, recreating and reassigning it to the new player
+/mob/dead/new_player/proc/reset_menu_hud()
+	set name = "Reset Lobby Menu HUD"
+	set category = "OOC"
+	var/mob/dead/new_player/new_player = usr
+	if(!COOLDOWN_FINISHED(new_player, reset_hud_cooldown))
+		to_chat(new_player, span_warning("You must wait <b>[DisplayTimeText(COOLDOWN_TIMELEFT(new_player, reset_hud_cooldown))]</b> before resetting the Lobby Menu HUD again!"))
+		return
+	if(new_player?.client)
+		COOLDOWN_START(new_player, reset_hud_cooldown, RESET_HUD_INTERVAL)
+		qdel(new_player.hud_used)
+		create_mob_hud()
+		hud_used.show_hud(hud_used.hud_version)
+		to_chat(new_player, span_info("Lobby Menu HUD reset. You may reset the HUD again in <b>[DisplayTimeText(RESET_HUD_INTERVAL)]</b>."))
+		return
```

Comment:
```suggestion
	if(!new_player?.client)
		return
	COOLDOWN_START(new_player, reset_hud_cooldown, RESET_HUD_INTERVAL)
	qdel(new_player.hud_used)
	create_mob_hud()
	hud_used.show_hud(hud_used.hud_version)
	to_chat(new_player, span_info("Lobby Menu HUD reset. You may reset the HUD again in <b>[DisplayTimeText(RESET_HUD_INTERVAL)]</b>."))
```
early returns

Reply:
fwiw the ending return was to prevent the verb from including the `undef` in its run

Reply:
that's uh, not how undef works. it's a compile time thing the proc doesn't know about it

Reply:
it did throw a warning though lol unless i went mad and indented it that one time, lemme check